### PR TITLE
Document individual search query string

### PR DESCRIPTION
**Pull request by Sicklou**
I added the information of how getting the individual search into the query string, since nothing was written in the doc.

### CODE REVIEW

File: packages/tables/docs/02-getting-started.md [hunk 1]
```diff
@@ -441,7 +441,8 @@ protected $queryString = [
     'tableFilters',
     'tableSortColumn',
     'tableSortDirection',
-    'tableSearchQuery' => ['except' => ''],
+    'tableSearchQuery' => ['except' => ''], // For global search
+    'tableColumnSearchQueries',  // For individual search
```
Comment (zepfietje) [inline packages/tables/docs/02-getting-started.md]:
```suggestion
    'tableSearchQuery' => ['except' => ''],
    'tableColumnSearchQueries',
```

File: packages/tables/docs/03-columns/01-getting-started.md [hunk 2]
```diff
@@ -175,6 +175,15 @@ use Filament\Tables\Columns\TextColumn;
 TextColumn::make('title')->searchable(isIndividual: true, isGlobal: false)
 ```
 
+If you want to have the search persisted in the query string, you have to add `tableColumnSearchQueries` to the `queryString` property :
```
Comment (zepfietje) [inline packages/tables/docs/03-columns/01-getting-started.md]:
```suggestion
You may optionally persist the search in the query string:
```

File: packages/tables/docs/03-columns/01-getting-started.md [hunk 3]
```diff
@@ -175,6 +175,15 @@ use Filament\Tables\Columns\TextColumn;
...
+
+```php
+    protected $queryString = [
+        ...,
+        'tableColumnSearchQueries',
+    ];
```
Comment (zepfietje) [inline packages/tables/docs/03-columns/01-getting-started.md]:
```suggestion
protected $queryString = [
    ...,
    'tableColumnSearchQueries',
];
```

**Review by zepfietje — CHANGES_REQUESTED** [review]

File: packages/tables/docs/03-columns/01-getting-started.md [hunk 4]
```diff
@@ -175,6 +175,15 @@ use Filament\Tables\Columns\TextColumn;
 TextColumn::make('title')->searchable(isIndividual: true, isGlobal: false)
 ```
 
+You may optionally persist the search in the query string:
+
+```php
+protected $queryString = [
+    ...,
```
Comment (zepfietje) [inline packages/tables/docs/03-columns/01-getting-started.md]:
```suggestion
    // ...
```

**Review by zepfietje — CHANGES_REQUESTED** [review]

**Review by zepfietje — APPROVED** [review]

**Comment by Sicklou** [comment]
Thanks for the review.

File: packages/tables/docs/03-columns/01-getting-started.md [hunk 5]
```diff
@@ -175,6 +175,15 @@ use Filament\Tables\Columns\TextColumn;
 TextColumn::make('title')->searchable(isIndividual: true, isGlobal: false)
 ```
 
+You may optionally persist the search in the query string:
```
Comment (danharrin) [inline packages/tables/docs/03-columns/01-getting-started.md]:
```suggestion
You may optionally persist the searches in the query string:
```

**Pull request merged**